src/api/v1/authorization/routes/logout.ts

```diff
@@ -0,0 +1,20 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { getSession, terminateSession } from '../../../../security/session'
+
+export default async (req: FastifyRequest < { Body: {} }>, res: FastifyReply): Promise<void> => {
```

Comment:
No need to explicitly type the body as empty if we don't care what's in it
```suggestion
export default async (req: FastifyRequest, res: FastifyReply): Promise<void> => {
```